modules/lib/typescript/index.d.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference path='global.d.ts' />
+
+declare const app: {
+    name: string;
+    version: string;
+    config: Record<string, string | undefined>;
+};
+
+declare const log: {
```

Comment:
There might be an opportunity to add a little bit of documentation regarding logging as JSDoc here.
Maybe some of the info from here: https://developer.enonic.com/docs/xp/stable/framework/globals#log